src/sass/_home.scss

```diff
@@ -0,0 +1,37 @@
+@use '_variables.scss' as *;
+
+.Home form {
+	position: relative;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin-top: 100px;
+	margin-bottom: 0;
+	.input {
+		display: flex;
+		flex-direction: column;
+		margin-right: 15px;
+	}
+	button::after {
+		position: absolute;
+		top: -9px;
+		right: -38px;
+		content: url('../../public/img/after.svg');
+	}
```

Comment:
Just learned files in the public directory are served at the root path.
Instead of /public/img/after.svg, we can just use /img/after.svg.

Reply:
neat!